hadrian.cabal

```diff
@@ -124,7 +124,7 @@ executable hadrian
                        , extra                >= 1.4.7
                        , mtl                  == 2.2.*
                        , QuickCheck           >= 2.6 && < 2.10
-                       , shake                >= 0.15.6
+                       , shake                == 0.16
```

Comment:
Suggest either >= 0.16 or 0.16.*. It should allow 0.16.1.

Reply:
Sure, thanks